tests/features/resumable-tasks.spec.ts

```diff
@@ -35,10 +35,15 @@ export function testResumableTasks(): void {
     let eventLog: EventLog;
     let eventStream: EventStream;
     let dwn: Dwn;
+    let consoleError: (message?: any, ...optionalParams: any[]) => void;;
```

Comment:
Unrelated to this PR, but didn't like seeing the error output in the testing console, so surpassed it during tests.

Reply:
No worries, you added something similar with `originalConsoleErrorFunction` for `event-stream.spec.ts`